### PR TITLE
Update ETC.tsv (through Real Time Translation)

### PR DESCRIPTION
**Pull request by Sunaries**
Minor updates in ETC. With the real time translation tool, I've made some appropriate changes.

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -6172,7 +6172,7 @@ ETC_20150317_006172	to move to the dungeon entrance. {nl} Be careful when you're
 ETC_20150317_006173	Gems can be obtained by hunting monsters and bosses in dungeons. {nl} Be careful, as the gems may be lost when you die.
 ETC_20150317_006174	Equipment
 ETC_20150317_006175	Sub-Weapon
-ETC_20150317_006176	Consumable
+ETC_20150317_006176	Consume
```
Comment (Soukyuu) [inline ETC.tsv]: What's the context of this change?

Comment (Soukyuu) [inline ETC.tsv]: This change should be reverted:
https://cdn.discordapp.com/attachments/80317349386780672/85134269197848576/consumable.png

Otherwise, we will be breaking the pattern on the categories.

File: ETC.tsv [hunk 2]
```diff
@@ -1462,17 +1462,17 @@ ETC_20150317_001461	Duration
 ETC_20150317_001462	User
 ETC_20150317_001463	Time Left
 ETC_20150317_001464	Name
-ETC_20150317_001465	{S18}{ol}{#ffffcc}Your name cannot exceed 16 characters {nl}including blank spaces.
+ETC_20150317_001465	{S18}{ol}{#ffffcc}Your name cannot exceed 16 characters {nl} including blank spaces.
```
Comment (Soukyuu) [inline ETC.tsv]: Why the space? A new line tag moves the text to the start of the line. Your edit would make it appear offset by one character
&nbsp;like this.

File: ETC.tsv [hunk 3]
```diff
@@ -1462,17 +1462,17 @@ ETC_20150317_001461	Duration
...
 ETC_20150317_001466	{S18}{ol}{#ffffcc}Disconnected from the server.{nl}
 ETC_20150317_001467	{S18}{ol}{#ffffcc}Move to the login mode after 3 seconds.
 ETC_20150317_001468	{S18}{ol}{#ffffcc}Cannot connect to server.
 ETC_20150317_001469	{S18}{ol}{#ffffcc}Your ID exceeds the maximum length allowed.
-ETC_20150317_001470	{S18}{ol}{#ffffcc}Your username or password is incorrect. {nl}Please note that passwords are case sensitive.
+ETC_20150317_001470	{S18}{ol}{#ffffcc}Your username or password is incorrect. {nl} Please note that passwords are case sensitive.
```
Comment (Soukyuu) [inline ETC.tsv]: Same as 1465

File: ETC.tsv [hunk 4]
```diff
@@ -1462,17 +1462,17 @@ ETC_20150317_001461	Duration
...
 ETC_20150317_001471	{S18}{ol}{#ffffcc}Server shutdown is not completed, or you are already logged in.{nl} Server shutdown may take up to 3 minutes.
 ETC_20150317_001472	{S18}{ol}{#ffffcc}Wrong verification code.
-ETC_20150317_001473	{S18}{ol}{#ffffcc}This name already exists. {nl}Please enter a different name.
-ETC_20150317_001474	{S18}{ol}{#ffffcc}Cannot create character.
-ETC_20150317_001475	{S18}{ol}{#ffffcc}Cannot delete character.
+ETC_20150317_001473	{S18}{ol}{#ffffcc}This name already exists. {nl} Please enter a different name.
```
Comment (Soukyuu) [inline ETC.tsv]: Another questionable space.

File: ETC.tsv [hunk 5]
```diff
@@ -1462,17 +1462,17 @@ ETC_20150317_001461	Duration
...
+ETC_20150317_001474	{S18}{ol}{#ffffcc}You cannot create a character
```
Comment (Soukyuu) [inline ETC.tsv]: Missing a dot at the end of the sentence.

File: ETC.tsv [hunk 6]
```diff
@@ -4340,7 +4340,7 @@ ETC_20150317_004339	Requires iCoin.{nl}Are you sure you want to reset?
 ETC_20150317_004340	{@st59} Increase in Physical Attack and Critical Attack{/}
 ETC_20150317_004341	{@st59} Increase in HP, HP Recovery rate, Critical Resistance and Max Inventory Slots{/}
 ETC_20150317_004342	{@st59} Increase in Magic Damage and Block Penetration{/}
-ETC_20150317_004343	{@st59} Increase in Accuracy, Evasion and Critical Attack {/}
+ETC_20150317_004343	{@st59} Increase in Accuracy, Evasion and Critical Rate {/}
```
Comment (Soukyuu) [inline ETC.tsv:4343]: That's already part of #871 
Please tell me when you include my fixes so that I don't keep a separate PR.

File: ETC.tsv [hunk 7]
```diff
@@ -1745,7 +1745,7 @@ ETC_20150317_001744	Not possible on this item.
 ETC_20150317_001745	This item is not equipped with a gem.
 ETC_20150317_001746	Please select a socket.
 ETC_20150317_001747	Show all
-ETC_20150317_001748	Consume
+ETC_20150317_001748	Consumable
```
Comment (Soukyuu) [inline ETC.tsv]: Line 6176 shows Consumable as the category for items in game, so this one should stay as "Consume".

File: ETC.tsv (hunk 1 above)
Comment (Sunaries) [inline ETC.tsv]: Corrected.

File: ETC.tsv (hunk 6 above)
Comment (Sunaries) [inline ETC.tsv:4343]: I forgot to tell you. This was because of the previous pull request that wasn't supposed to be there. I was afraid imc would override your change.

File: ETC.tsv (hunk 5 above)
Comment (Sunaries) [inline ETC.tsv]: Corrected.

**Comment by imcgames** [comment]
Thank you :+1:
